src/librustc_unicode/u_str.rs

```diff
@@ -27,9 +27,16 @@ use core::str::Split;
 
 use tables::grapheme::GraphemeCat;
 
-/// An iterator over the words of a string, separated by a sequence of whitespace
-#[stable(feature = "rust1", since = "1.0.0")]
-pub struct Words<'a> {
+#[deprecated(reason = "struct Words is being replaced by struct SplitWhitespace",
+             since = "1.0.0")]
```

Comment:
If we’re landing this for 1.1.0 then my previous comment about stable→unstable is moreso important, because now:

* A while ago: unstable
* 1.0 beta2: stable
* 1.0: stable
* 1.1: unstable + deprecated

which makes no sense, even if it was made stable by mistake.